### PR TITLE
Creates an independent default workflow store.

### LINKED ISSUE #236: Workflow id reverts to default value, even when assignment is active.
https://github.com/zooniverse/wildcam-gorongosa-education/issues/283

### PR DESCRIPTION
**Pull request by simoneduca**
Field guide uses the default store.
Fix #236.

### CODE REVIEW

**Comment by simoneduca** [comment]
Pinging @rogerhutchings too, who suggested this fix, for extra eyes 👀

File: app/pages/field-guide.cjsx [hunk 1]
```diff
@@ -6,7 +6,9 @@ _ = require 'lodash'
 
 ProjectMetadata = require '../partials/project-metadata'
 
-workflowStore = require '../stores/workflow-store'
+# Uses default-workflow-store to ensure the default workshop is always set,
+# without interfering with assignments workflows
+workflowStore = require '../stores/default-workflow-store'
```
Comment (rogerhutchings) [inline app/pages/field-guide.cjsx]: I'd rename this to `defaultWorkflowStore` to make it explicit. You'll need to rename it on line 22 (I think) as well

File: app/stores/default-workflow-store.coffee [hunk 2]
```diff
@@ -0,0 +1,17 @@
+Reflux = require 'reflux'
+{api} = require '../api/client'
+config = require '../lib/config'
+
+
```
Comment (rogerhutchings) [inline app/stores/default-workflow-store.coffee]: Remove all but one blank line

File: app/stores/default-workflow-store.coffee [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+module.exports = Reflux.createStore
+  data: null
+
+  init: ->
+    api.type('workflows').get(config.workflowId)
+      .then (@data) =>
+        console.warn "Default workflow #{@data.id}"
```
Comment (rogerhutchings) [inline app/stores/default-workflow-store.coffee]: Remove `console.warn`

**Comment by shaunanoordin** [comment]
- Tested on localhost:3434 with OSX+Chrome52, using modified Assignments page link from lab.wildcamgorongosa.org
- As a Student (`zootester1`) from lab.wildcamgorongosa.org, if I access an Assignment link (e.g. http://localhost:3434/#/classify/assignment-32/access_token=...), I will be taken to WildCam Gorongosa's Classify page, with the correct Assignment selected, and am logged in if previously not logged in.
  - Functions correctly if Student has already previously logged into WildCam Gorongosa as him/herself. Not tested if, e.g. Student comes from Lab as `zootester1` but already logged in as `zootester2` at WildCam Gorongosa.
- Confirmed via monitoring Network traffic that the error in #236 does not reoccur - two workflow IDs (the default and the assignment) are NOT simultaneously fetched. ONLY the Assignment ID is requested.
- Confirmed that base functionality appears OK:
  - As a non-logged-in user, I am able to Classify items and access the Field Guide (including specific URLs to a specific animal).
  - As a logged-in user with no assignments, I am able to Classify items and access the Field Guide (including specific URLs to a specific animal).
  - As a logged-in user with a specific assignment, while working on an Assignment, I am able to access the Field Guide. (Without the change in workflow IDs causing any shenanigans.)

Status: LGTM
Actions: Merging

**Pull request merged**